docs/specs/metadata.yml

```diff
@@ -57,6 +57,26 @@ outputs:
     { "redirectionUrl":"/sccm/mdm/understand/hybrid-mobile-device-management" }
   build.manifest:
 ---
+# generate same document id and version id with 'redirection with document id' evem it's out of build scope
```

Comment:
evem -> even

I think we really need an automated spellchecker that integrate with GH PR

Reply:
how about auto check extension within VS?

---
In reply to: [214230742](https://github.com/dotnet/docfx/pull/3338#discussion_r214230742) [](ancestors = 214230742)